photon-core/src/main/java/org/photonvision/vision/processes/VisionModule.java

```diff
@@ -483,16 +481,6 @@ public void setCameraNickname(String newName) {
         inputFrameSaver.updateCameraNickname(newName);
         outputFrameSaver.updateCameraNickname(newName);
 
-        // Rename streams
-        streamResultConsumers.clear();
-
-        // Teardown and recreate streams
-        destroyStreams();
```

Comment:
So does this mean we don't actually ever recreate the MJPGFrameConsumer now? That shouldn't cause issues (I guess the metadata about camera name in NT might be wrong unless we tell it to change? But AFAIK nobody uses that anyways)

Reply:
The camera name isn't used in the `MJPGFrameConsumer` constructor, so nothing should change afaik